modape/modis/window.py

```diff
@@ -218,15 +218,18 @@ def generate_mosaics(self,
                 "outputType": attrs["dtype"],
                 "noData": attrs["nodata"],
                 "outputSRS": target_srs,
+                "projWin": aoi
             }
 
-            if aoi is not None:
-                translate_options.update(
-                    {"projWin": aoi}
-                )
-
             translate_options.update(kwargs)
 
+            if aoi is not None and any(output_res):
```

Comment:
Should we be using `all` instead of `any`? 

I think we can't arrive here with a list that contains one `None` element, but still the calculations below _require_ two elements which are not `None`.

Reply:
Agree; just pushed the change